src/Microsoft.AspNet.Routing/Template/TemplateRoute.cs

```diff
@@ -202,26 +202,20 @@ public virtual string GetVirtualPath(VirtualPathContext context)
 
             // Validate that the target can accept these values.
             var childContext = CreateChildVirtualPathContext(context, values.AcceptedValues);
+
             var path = _target.GetVirtualPath(childContext);
             if (path != null)
             {
                 // If the target generates a value then that can short circuit.
-                context.IsBound = true;
```

Comment:
`context.IsBound` means that the values are 'validated' since we're inspecting the values we never set them to validated. 

This is the difference between a 'match' and a 'fallback match' in the new system.